hack/lib/mesh_resources/smcp.yaml

```diff
@@ -7,6 +7,11 @@ spec:
   profiles:
   - default
   proxy:
+    networking:
+      trafficControl:
+        inbound:
+          excludedPorts:
+          - 15020
```

Comment:
Can we add port 8444 here for metrics to work?

Reply:
Thank you @skonto I added `8444`.
By the way, do you have an idea how to stop alert error - [alerts.json](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-knative_serverless-operator/1085/pull-ci-openshift-knative-serverless-operator-main-4.7-upstream-e2e-aws-ocp-47/1412963876953133056/artifacts/upstream-e2e-aws-ocp-47/serverless-e2e/artifacts/build-Hddbxk3E/alerts.json).
We need to set  `metrics.backend-destination` to `"none"` like https://github.com/openshift-knative/serverless-operator/pull/1014 ?

Reply:
Nope we should have the target as is. The workaround that exposes the port should make things work.